code/game/gamemodes/shadowling/shadowling_abilities.dm

```diff
@@ -607,12 +607,12 @@ datum/reagent/shadowling_blindness_smoke //Reagent used for above spell
 				playsound(thrallToRevive, 'sound/machines/defib_zap.ogg', 50, 1)
 				user.Beam(thrallToRevive,icon_state="red_lightning",icon='icons/effects/effects.dmi',time=1)
 				sleep(10)
-				thrallToRevive.revive()
-				thrallToRevive.visible_message("<span class='boldannounce'>[thrallToRevive] heaves in breath, dim red light shining in their eyes.</span>", \
+				if(thrallToRevive.revive(full_heal = 1))
```

Comment:
this will make it so that if we try to revive a thrall with its brain missing, we won't get the success messages since the ressuscitation failed.